src/Illuminate/Support/Arr.php

```diff
@@ -294,11 +294,11 @@ public static function get($array, $key, $default = null)
         }
 
         foreach (explode('.', $key) as $segment) {
-            if (static::accessible($array) && static::exists($array, $segment)) {
-                $array = $array[$segment];
-            } else {
-                return value($default);
+            if (!static::accessible($array) && !static::exists($array, $segment)) {
```

Comment:
I believe you made a mistake here. The code you are looking for is:

```
if (! static::accessible($array) ||  ! static::exists($array, $segment)) {
```

Reply:
Uh... I missed to turn the `and` to an `or`!